src/msac.rs

```diff
@@ -149,10 +149,43 @@ impl Default for Rav1dMsacDSPContext {
 
 pub type EcWin = usize;
 
+#[repr(C)]
+struct MsacAsmContextBuf {
```

Comment:
Document what this must point to hold the unsafe impl invariants. Either these must both be null or they must point into the `data` field of a `MsacContext` that also contains this structure.